go/vt/vtgate/planbuilder/rewrite.go

```diff
@@ -28,16 +29,17 @@ type (
 		proceed   bool
 		tblColMap map[*sqlparser.AliasedTableExpr]sqlparser.SelectExprs
 	}
-	starRewriter struct {
-		err      error
-		semTable *semantics.SemTable
+	rewriter struct {
```

Comment:
I don't love this name. we already have other rewriters in the system - what makes this _the_ rewriter?

Reply:
@systay, right, nothing makes it _the_ one and only rewriter. How about `postSemanticAnalysisRewriter` or something similar?